contributors.json

```diff
@@ -718,4 +718,11 @@
     "country": "India",
     "name": "Aarushi"
   }
+  "shashinkaliyanage": {
```

Comment:
Let's fix the JSON syntax and order the values in ascending order by Github user name